### PR TITLE
Add Mobile Deep Learning book

### PR DESCRIPTION
**Pull request by mheriyanto**

### CODE REVIEW

**Comment by margaretmz** [comment]
Thanks for the PR, @mheriyanto! Great book @xprilion & @Rimjhim28.

**Comment by mheriyanto** [comment]
> Thanks for the PR, @mheriyanto! Great book @xprilion & @Rimjhim28.

You're welcome @margaretmz.

File: README.md [hunk 1]
```diff
@@ -164,6 +164,7 @@ Interested but not sure how to get started? Here are some learning resources tha
 * 2018-03-30 - [Using TensorFlow Lite on Android](https://blog.tensorflow.org/2018/03/using-tensorflow-lite-on-android.html)) - By Laurence Moroney.
 
 ### Books
+* 2020-04-06 [Mobile Deep Learning with TensorFlow Lite, ML Kit and Flutter](https://www.packtpub.com/product/mobile-deep-learning-with-tensorflow-lite-ml-kit-and-flutter/9781789611212): Build scalable real-world projects to implement end-to-end neural networks on Android and iOS ([GitHub](https://github.com/PacktPublishing/Mobile-Deep-Learning-Projects)) - By Anubhav Singh ([@xprilion](https://github.com/xprilion)) and Rimjhim Bhadan ([@Rimjhim28](https://github.com/Rimjhim28)).
```
Comment (xprilion) [inline README.md:167]: Rimjhim's name is incorrect here. It should be `Rimjhim Bhadani`.

**Review by xprilion — COMMENTED** [review]
Kindly fix the name spelling :)

**Comment by xprilion** [comment]
Thank you @mheriyanto and @margaretmz for adding the book to the resources 🙌

**Comment by margaretmz** [comment]
Fixed the author name Rimjhim Bhadani. Thanks for catching the error.

**Pull request merged**